src/hmem_cuda.c

```diff
@@ -132,6 +142,23 @@ int cuda_copy_from_dev(uint64_t device, void *host, const void *dev, size_t size
 	return -FI_EIO;
 }
 
+int ofi_hmem_cuda_dev_register(struct fi_mr_attr *mr_attr, uint64_t *handle)
+{
+	if (hmem_cuda_use_gdrcopy)
+		return cuda_gdrcopy_dev_register(mr_attr, handle);
+
+	*handle = mr_attr->device.cuda;
+	return 0;
```

Comment:
FI_SUCCESS is used throughout this file instead of 0. Can it be used here as well?

Reply:
Thanks! fixed in new update.